tests/testthat/test-ensembleMethods.R

```diff
@@ -30,22 +29,21 @@ test_that("We can get variable importance in ensembles", {
   expect_is(varImp(ens.reg, scale = TRUE, weight = TRUE), "data.frame")
 })
 
-test_that("We get warnings when scale is set to FALSE and weight is TRUE", {
-  skip_on_cran()
+test_that("varImp works for caretEnsembles", {
```

Comment:
made this test easier to follow with a loop (instead of the unrolled loop), and checks for the rows/cols at each loop iteration.